test/unit/spec/PwdResetEmailSent_spec.js

```diff
@@ -0,0 +1,40 @@
+/* eslint max-params: [2, 18], max-statements: 0 */
+define([
+  'q',
+  'okta',
+  'helpers/mocks/Util',
+  'helpers/dom/Form',
+  'LoginRouter',
+  'sandbox',
+  'helpers/util/Expect'
+],
+function (Q, Okta, Util, Form, Router, $sandbox, Expect) {
+
+  var {_} = Okta;
+  var itp = Expect.itp;
+
+  function setup (settings) {
+    var router = new Router(_.extend({
+      el: $sandbox,
+      baseUrl: 'https://foo.com'
+    }, settings));
+    var form = new Form($sandbox);
+    Util.registerRouter(router);
+    Util.mockRouterNavigate(router);
+    router.recoveryEmailSent();
+    return Expect.waitForPwdResetEmailSent({
+      router: router,
+      form: form
+    });
+  }
+
+  Expect.describe('check back link', function () {
+    itp('does not show back link if hideBackToSignInForReset is true', function () {
```

Comment:
Can you add a test to test that it does show back link when hideBackToSignInForReset is false?

Reply:
Sure, although we don't have in other spec files these 2 tests for that

Reply:
We should probably add that test there too then